src/components/sidebar/components/sidebar-nav-menu-item/index.tsx

```diff
@@ -2,57 +2,51 @@ import { KeyboardEventHandler, useEffect, useRef, useState } from 'react'
 import Link from 'next/link'
 import { IconChevronRight16 } from '@hashicorp/flight-icons/svg-react/chevron-right-16'
 import { IconExternalLink16 } from '@hashicorp/flight-icons/svg-react/external-link-16'
-import MaybeInternalLink from 'components/maybe-internal-link'
+import isAbsoluteUrl from 'lib/is-absolute-url'
 import { MenuItem } from 'components/sidebar'
 import {
   SidebarHorizontalRule,
   SidebarSectionHeading,
 } from 'components/sidebar/components'
+import Text from 'components/text'
+import { SidebarNavMenuItemProps } from './types'
 import s from './sidebar-nav-menu-item.module.css'
 
-interface SidebarMenuItemProps {
-  item: MenuItem
-}
-
-const SidebarNavLink = ({ item }) => {
-  if (item.fullPath) {
-    return (
-      <li>
-        <Link href={item.fullPath}>
-          <a
-            aria-current={item.isActive ? 'page' : undefined}
-            className={s.sidebarNavMenuItem}
-            // TODO: this might break some accessible labels, probably need aria-label
-            dangerouslySetInnerHTML={{ __html: item.title }}
-          />
-        </Link>
-      </li>
-    )
-  }
+/**
+ * Handles rendering a link menu item in the Sidebar. Will automatically
+ * determine whether or not the link is external to DevDot, and will render an
+ * external link icon if the link is external.
```

Comment:
I believe this "All Tutorials" link is external because it will be linking to Learn for beta. I've created a ticket for clearing this up with design and also for figuring out how we want external links to behave when clicked. Ticket here: https://app.asana.com/0/1202110981600689/1202129912162157/f.